WordPress/Classes/Networking/LoginClient.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import WordPressAPI
+import AutomatticTracks
+
+actor LoginClient {
```

Comment:
I don't see the need to make this an actor type, because there is no data race to prevent?

Reply:
It doesn't at the moment, but I wanted to require async access to it from the start instead of marking all the methods `async`. 

Might not be worth it, but we could roll it back if needed? I'd say better to make the decision after we've used it a bit in https://github.com/wordpress-mobile/WordPress-iOS/tree/add/application-token-management?